float64 fix_depth
std_msgs/String user
---
bool success
